client/src/components/ProjectsTextfields/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Fragment, Component } from 'react';
+import { TextField, Button } from '@material-ui/core';
+import { withStyles } from '@material-ui/styles';
+import style from './style';
+import axios from 'axios';
+
+class Textfields extends Component {
+	state = {
+		firstProject: '',
+		secondProject: '',
+		userId: this.props.userId,
+		Submitted: false,
+	};
+	handleClick = (event) => {
+		axios
+			.post('/api/user/:userId/projects', {
+				firstProject: this.state.firstProject,
+				secondProject: this.state.secondProject,
+				userId: this.state.userId,
+			})
+			.then((data) => {
```

Comment:
could you use destruction ({data}) to the data to avoid recursion like that data.data